lib/ukatomic/include/uk/atomic.h

```diff
@@ -27,83 +27,81 @@
  * SUCH DAMAGE.
  */
 
-#ifndef __UKARCH_ATOMIC_H__
-#define __UKARCH_ATOMIC_H__
+#ifndef __UK_ATOMIC_H__
+#define __UK_ATOMIC_H__
 
 #include <uk/arch/lcpu.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#include <uk/asm/atomic.h>
-
 /**
  * Perform an atomic load operation.
  */
-#define ukarch_load_n(src) \
+#define uk_load_n(src) \
 	__atomic_load_n(src, __ATOMIC_SEQ_CST)
 
 /**
  * Perform an atomic store operation.
  */
-#define ukarch_store_n(src, value) \
+#define uk_store_n(src, value) \
 	__atomic_store_n(src, value, __ATOMIC_SEQ_CST)
 
 /**
  * Perform an atomic fetch and add/sub operation.
  */
-#define ukarch_fetch_add(src, value) \
+#define uk_fetch_add(src, value) \
 	__atomic_fetch_add(src, value, __ATOMIC_SEQ_CST)
-#define ukarch_fetch_sub(src, value) \
+#define uk_fetch_sub(src, value) \
 	__atomic_fetch_sub(src, value, __ATOMIC_SEQ_CST)
 
 /**
  * Perform an atomic add/sub and fetch operation.
  */
-#define ukarch_add_fetch(src, value) \
+#define uk_add_fetch(src, value) \
 	__atomic_add_fetch(src, value, __ATOMIC_SEQ_CST)
-#define ukarch_sub_fetch(src, value) \
+#define uk_sub_fetch(src, value) \
 	__atomic_sub_fetch(src, value, __ATOMIC_SEQ_CST)
 
 /**
  * Perform an atomic increment/decrement operation and return the
  * previous value.
  */
-#define ukarch_inc(src) \
-	ukarch_fetch_add(src, 1)
-#define ukarch_dec(src) \
-	ukarch_fetch_sub(src, 1)
+#define uk_inc(src) \
```

Comment:
There is one instance of `ukarch_inc` remaining in `unixsock.c`:
```
lib/posix-unixsocket/unixsock.c:		if (!ukarch_inc(&data->listen.count))
```
Perhaps it was added after you did this work.